docs/_user_guide/best_practices_optimization.md

```diff
@@ -309,179 +357,176 @@ We point to the AMPL floating network license with `AMPL(Environment("/software/
 --env GUROBI_HOME /software/non-free/Gurobi/gurobi1000/linux64 \
 --env GRB_LICENSE_FILE /software/non-free/Gurobi/gurobi1000/linux64/gurobi.lic \
---env KNITRODIR /software/non-free/knitro-12.1.1/knitro-12.1.1-Linux-64 \
---env ARTELYS_LICENSE_NETWORK_ADDR license4.stanford.edu \
---env LD_LIBRARY_PATH '/software/non-free/knitro-12.1.1/knitro-12.1.1-Linux-64/lib:/software/non-free/Gurobi/gurobi1000/linux64/lib:${LD_LIBRARY_PATH}' \
+--env KNITRODIR /software/non-free/knitro/14.0.0 \
+--env ARTELYS_LICENSE_NETWORK_ADDR srcc-license-srcf.stanford.edu \
+--env LD_LIBRARY_PATH '/software/non-free/knitro/14.0.0/lib:${LD_LIBRARY_PATH}' \
 --env PATH '/software/non-free/Gurobi/gurobi1000/linux64/bin:${PATH}' \
---env PYTHONPATH '/software/non-free/knitro-12.1.1/knitro-12.1.1-Linux-64/examples/Python:${PYTHONPATH}'
+--env PYTHONPATH '/software/non-free/knitro/14.0.0/examples/Python:${PYTHONPATH}'
 ```
 
-On <a href="/gettingStarted/8_jupyterhub.html" target="_blank">JupyterHub</a>, launch the new `opt` kernel 
-and test the package imports for `amplpy`, `gurobipy` and all other previous imports.
+On [JupyterHub](/_getting_started/jupyter/){:target="_blank"}, launch the new `opt_combined` kernel and test the package imports for `amplpy`, `gurobipy` and all other previous imports.
+
+!!! Note
+    Since we already have a Jupyter kernel named `opt`, we chose a different name `opt_combined` for the combined Gurobi, AMPL and Knitro kernel. If you choose the same kernel name `opt`, the `opt` kernel will be overwritten to reference the latest kernel install from the active environment.
 
-**Note**: If already have a Jupyter kernel named `opt`, choose a different name for the combined Gurobi, AMPL and Knitro kernel. Otherwise, the `opt` kernel will be overwritten to reference the latest kernel install from the active environment.
+## Running Software on Slurm
 
-## Running Batch Jobs with Slurm
-The Yen-Slurm cluster is comprised of 7 shared compute nodes that use <a href="https://slurm.schedmd.com/documentation.html" target="_blank">Slurm</a> to schedule jobs and manage a queue of resources
-(if there are more requests than resouces available). It is a batch submission environment like the
-<a href="https://www.sherlock.stanford.edu" target="_blank">Sherlock HPC</a> cluster.
- 
-We can use the same `opt` virtual environment to run python code on the Slurm nodes. 
+### Running Batch Jobs 
 
-We load the optimization software modules, activate the virtual python environment before calling `python` in
-the slurm script. Let's save this slurm script to a file named `test-opt.slurm`:
+The Yen-Slurm cluster is comprised of 11 shared compute nodes that use [Slurm](https://slurm.schedmd.com/documentation.html){:target="_blank"} to schedule jobs and manage a queue of resources (if there are more requests than resources available). It is a batch submission environment like the [Sherlock HPC](https://www.sherlock.stanford.edu){:target="_blank"} cluster.
```

Comment:
We can cut this info and just point to the slurm page on rcpedia

Reply:
I actually added the link to the Yen Slurm page at the end of the page. To make this change, I deleted the last sentence: 

For more information on using Yen Slurm, visit [this page](/_user_guide/slurm/){:target="_blank"} 

deleted this paragraph, and modified the next paragraph by adding a link: 

To use the optimization software on the [Yen-Slurm cluster](/_user_guide/slurm/){:target="_blank"}, we first need to load the required modules and activate the virtual Python environment before calling python in the Slurm script.

Does this change look good to you?

Reply:
yes, looks good